### PR TITLE
Added color picker for input

### PR DESCRIPTION
**Pull request by nishchay17**
For better user experience I switch text-based color input to color-picker input.

### CODE REVIEW

**Comment by vishnuramana** [comment]
Hey ! Thanks for the PR. I will review it and get back.

File: src/App.js [hunk 1]
```diff
@@ -16,11 +16,11 @@ class App extends Component {
         borderColor: "#2e2e2e",
         baseColor: "#17a2b8",
         centerColor: "#459cff",
-        centerBorderColor: "#fff",
+        centerBorderColor: "#ffffff",
         handColors: {
           second: "#d81c7a",
-          minute: "#fff",
-          hour: "#fff"
+          minute: "#ffffff",
```
Comment (vishnuramana) [inline src/App.js:22]: Any significance for this change ?

File: src/Form.js [hunk 2]
```diff
@@ -67,11 +67,11 @@ class Form extends Component {
         }
         return colorCode;
     }
-
-
+    
     render() {
         return (
             <form className="col-12 form">
+                {console.log( this.state.options.borderColor)}
```
Comment (vishnuramana) [inline src/Form.js]: Can you please remove this ?

Comment (nishchay17) [reply src/Form.js]: yeah

File: src/App.js (hunk 1 above)
Comment (nishchay17) [reply src/App.js:22]: The value in color input has to be of length 7, so the 4 length will not work.
[https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/color#value](ref)

**Comment by vishnuramana** [comment]
Looks good. Merging. Thanks for the PR.

**Pull request merged**